hack/build.sh

```diff
@@ -74,6 +74,7 @@ if (echo "${TAGS}" | grep -q 'libvirt')
 then
 	export CGO_ENABLED=1
 fi
+export CGO_ENABLED=1
```

Comment:
If this is going to be effectively an always required setting, then the previous block could be removed